app/components/fastboot-only/body-bottom.js

```diff
@@ -47,6 +49,13 @@ export default Component.extend({
 		}, simpleStore));
 	}),
 
+	asyncScriptsPath: computed(function () {
+		const langPath = getLanguageCodeFromRequest(this.get('fastboot.request')),
```

Comment:
one const per var -
should we enable this in eslint?
maybe it can fix it automatically?

Reply:
I can create a ticket to enable it and fix it.